test/extended/deployments/deployments.go

```diff
@@ -152,7 +152,8 @@ var _ = g.Describe("deploymentconfigs", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			g.By(fmt.Sprintf("by checking that the second deployment exists"))
-			err = wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
+			// TODO when #11016 gets fixed this can be reverted to 30seconds
+			err = wait.PollImmediate(500*time.Millisecond, 1*time.Minute, func() (bool, error) {
```

Comment:
A bunch of the deployment tests have bumped timeouts, not sure if a single revert is enough. I would opt for a follow-up issue to  #10228 where we should revisit times in all deployment tests.